sycl/test-e2e/Graph/RecordReplay/record_inconsistent_contexts.cpp

```diff
@@ -0,0 +1,27 @@
+// REQUIRES: level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out 2>&1 | FileCheck %s
+//
+// CHECK: begin_recording called for a queue whose context differs from the graph context
+
+// This test check that an expection is thrown when we try to
+// record a graph whose context differs from the queue context.
+
+#include "../graph_common.hpp"
+
+int main() {
+  queue Queue;
+
+  property_list Properties{property::queue::in_order()};
+  context InOrderContext(Properties);
```

Comment:
Why are passing queue properties to the context?

Reply:
I wanted to be sure to have a different context. But you are right, we do not need it. I removed it in the update.